awx/main/isolated/manager.py

```diff
@@ -304,14 +303,9 @@ def consume_events(self, dispatcher):
                         # again on the next sync
                         pass
                     event_data.setdefault(self.event_data_key, self.instance.id)
```

Comment:
you deleted `event_data.setdefault(self.event_data_key, self.instance.id)` on line 436.

I believe you are free to do the same here.